.rubocop.yml

```diff
@@ -1,5 +1,12 @@
 inherit_from: .rubocop_todo.yml
 
+AllCops:
+  TargetRubyVersion: 1.9
```

Comment:
> users may let RuboCop know the oldest version of Ruby which your project supports with:

From
https://github.com/bbatsov/rubocop/blob/master/manual/configuration.md#setting-the-target-ruby-version

Alternative is to use a .ruby-version